### PR TITLE
feat: Copy existing task definition tags

### PR DESCRIPTION
**Pull request by nitrocode**
Adding tags in the describe_task_definition should give us the tags for registering new task definitions

Closes https://github.com/fabfuel/ecs-deploy/issues/130

### CODE REVIEW

File: ecs_deploy/ecs.py [hunk 1]
```diff
@@ -595,7 +595,6 @@ def update_task_definition(self, task_definition):
             volumes=task_definition.volumes,
             role_arn=task_definition.role_arn,
             execution_role_arn=task_definition.execution_role_arn,
-            tags=task_definition.tags,
```
Comment (fabfuel) [inline ecs_deploy/ecs.py]: Hi @nitrocode, thanks for your PR!
Isn't this needed to keep the tags in the new task definition?

Best
Fabian

Comment (nitrocode) [reply ecs_deploy/ecs.py]: Hi! Thanks for looking at it ☺️.

I think you're right. I'll step through this to make sure tags are passed appropriately

**Comment by nitrocode** [comment]
Hi @fabfuel , would you mind checking this PR again when you're free? I believe I fixed all the issues with it.

**Review by fabfuel — APPROVED** [review]
Looks good, thanks a lot @nitrocode !
I will merge and release the next days.

Best
Fabian

**Comment by nitrocode** [comment]
Thanks @fabfuel. Looking forward to it! :smile:

**Comment by nitrocode** [comment]
@fabfuel thanks for merging.

Will there be a new release?

**Comment by fabfuel** [comment]
Hi @nitrocode! I just created the release and pushed it to PyPI. Sorry for the delay - very busy weeks... 😅 
https://pypi.org/project/ecs-deploy/

Thanks again for your contribution - all the best!
Fabian

**Pull request merged**